### PR TITLE
Issue 7

### PR DESCRIPTION
**Pull request by DanSwtmn**
In the event tests, there is a commented out test that is not working for some reason even though I can get a response from testing the endpoint manually. Also with the updateEventById() I believe that you should be able to edit all fields and not only specific ones.

### CODE REVIEW

File: app/routes/event.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  authenticationMiddleware.verifyProfessor,
+  eventController.createEvent
+);
+
+// UPDATE a specific event
+router.patch(
+  '/',
+  authenticationMiddleware.validateAuthenticationAttachUser,
+  authenticationMiddleware.verifyProfessor,
+  eventController.createEvent
```
Comment (ShaydeNofziger) [inline app/routes/event.js]: Should this be `updateEvent`?

File: migrations/20160912145044_setup.js [hunk 2]
```diff
@@ -37,10 +37,18 @@ exports.up = knex =>
     table.integer('initial_resources').notNullable();
     table.integer('initial_mindset').notNullable();
     table.boolean('disabled').defaultsTo(false);
+  })
+  .createTable('event', (table) => {
+    table.increments('id');
+    table.string('name').unique().notNullable();
+    table.string('description').notNullable();
+    table.integer('default_damage').notNullable();
+    table.boolean('disabled').notNullable();
```
Comment (ShaydeNofziger) [inline migrations/20160912145044_setup.js]: To match the data model schema, (bool not required) we should default this to `false`.

**Review by ShaydeNofziger — CHANGES_REQUESTED** [review]

File: migrations/20160912145044_setup.js (hunk 2 above)
Comment (DanSwtmn) [reply migrations/20160912145044_setup.js]: Changed

File: app/routes/event.js (hunk 1 above)
Comment (DanSwtmn) [reply app/routes/event.js]: Yes you are correct i did change it

**Review by ShaydeNofziger — APPROVED** [review]

**Pull request merged**